tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -70,5 +97,16 @@ private static string ReadEnvironmentVariable(string key)
 
             return null;
         }
+
+        private static bool ReadBooleanEnvironmentVariable(string key, bool defaultValue)
+        {
+            var value = ReadEnvironmentVariable(key);
+            return value switch
+            {
+                null => defaultValue,
+                "1" or "true" or "True" or "TRUE" => true,
+                _ => false
+            };
```

Comment:
This read boolean setting is different from the "read boolean" setting we use with the rest of our configuration:

https://github.com/DataDog/dd-trace-dotnet/blob/7488004eef7732be5a53354b0c8e5c798ce9ca2d/tracer/src/Datadog.Trace/ExtensionMethods/StringExtensions.cs#L38

That said, on the native side, we calculate the value a different way: https://github.com/DataDog/dd-trace-dotnet/blob/7488004eef7732be5a53354b0c8e5c798ce9ca2d/tracer/src/Datadog.Trace.ClrProfiler.Native/environment_variables_util.h#L26-L44

IMO, in this case, we should match the _native_ implementation. The suggestion doesn't quite do that, but it's closer, and seems like a better trade off to me? 

```suggestion
            return value switch
            {
                "1" or "true" or "True" or "TRUE" => true,
                "0" or "false" or "False" or "FALSE" => false,
                _ => defaultValue
            };
```

Reply:
Fixed in 18c25b94dbc2afc43c86450a638b85de37b6d58e